base/setup/lib/setuplib.c

```diff
@@ -477,7 +477,7 @@ GetSourcePaths(
 
     /* Check whether the resolved \SystemRoot is a prefix of the image file path */
     // FIXME: commented out to allow installation from USB
-    // if (RtlPrefixUnicodeString(&SystemRootPath, InstallSourcePath, TRUE))
+    /**/if (RtlPrefixUnicodeString(&SystemRootPath, InstallSourcePath, TRUE))/**/
```

Comment:
I would encourage you to undo that change.
```suggestion
    // if (RtlPrefixUnicodeString(&SystemRootPath, InstallSourcePath, TRUE))
```

Reply:
This is from the 1st temporary commit, will go away in the final PR version.